Link Browser Pages: document configuration option pageIdSelector (#826)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst

in class :php:`\TYPO3\CMS\Recordlist\LinkHandler\RecordLinkHandler`
which does not contain Hooks or Slots.

> Special: Activate page UID field for directly entering a target page

Reviewer: I find the word "special" strange. can we just remove it?

--------------------------------------------------------------------

.. important::

>    This only works for the page LinkHandler. Not for custom added LinkHandler configurations.

Reviewer 2: "PageLinkHandler"

>    This only works for the page LinkHandler. Not for custom added LinkHandler configurations.

Reviewer: ```suggestion
   This only works for the page LinkHandler and **not** for custom added LinkHandler configurations.
```

> It is possible to activate an additional field in the link browser to enter the UID of a page

Reviewer 2: possible to enable an additional

directly instead of selecting it from the page tree. This is especially useful if you have a

> potentially large page tree in your TYPO3 project.

Reviewer: ```suggestion
potentially large page tree in your TYPO3 project and you know the the id of the to be linked page already.
```

.. figure:: ../../../Images/LinkBrowserTSConfigExamplepageIdSelector.png
   :alt: The link browser field for entering a page uid

> Activate the field with exactly the following User-/PageTSConfig::

Reviewer 2: Enable the field

> Activate the field with exactly the following User-/PageTSConfig::

Reviewer: ```suggestion
Activate the field with the following User-/PageTSConfig:
.. code-block:: typoscript

```
